app/models/user.rb

```diff
@@ -70,14 +70,18 @@ class User < ApplicationRecord
   has_many :wellplates, through: :collections
   has_many :screens, through: :collections
   has_many :research_plans, through: :collections
+  has_many :vessels, through: :collections
+
+  # created vessels will be kept when the creator goes (dependent: nil).
+  has_many :created_vessels, class_name: 'Vessel', inverse_of: :creator, dependent: nil
```

Comment:
Not sure about the dependent option. 
We most probably don't want `dependent(:delete)` as it would destroy all vessels in case their creator ever gets deleted. 

We might want to set `dependent: :nullify` so the created_vessels user_id will be set to null, potentially avoiding  user-not-found  issues, potentially creating user-is-nil issues. 

I decided for `dependent: nil` as this does nothing hence is the least invasive option.